windows/test_msi.ps1

```diff
@@ -48,27 +65,34 @@ if ($UPGRADE -eq "true")
         echo "❌ Failed installing latest version of the msi"
         exit 1
     }
-    echo "✅ Installation for ${PKG_LATEST_NAME} succeeded"
+    echo "✅ Installation for ${PKG_UPSTREAM_NAME} succeeded"
 }
 
-$version = $TAG -replace "v", ""
+$version = "$TAG" -replace "v", ""
 if ($PKG_DIR -eq "")
 {
     $PKG_DIR = "src\github.com\newrelic\${INTEGRATION}\build\package\windows\nri-${ARCH}-installer\bin\Release"
 }
 if ($PKG_NAME -eq "")
 {
-    $PKG_NAME = "${INTEGRATION}-${ARCH}.${version}.msi"
+    if ($PKG_TYPE -eq "msi")
+    {
+        $PKG_NAME = "${INTEGRATION}-${ARCH}.${version}.msi"
+    }
+    else
+    {
+        $PKG_NAME = "${INTEGRATION}-${ARCH}-installer.${version}.exe"
+    }
 }
-$PKG_LATEST_NAME = "${PKG_DIR}\${PKG_NAME}"
-write-host "::group::ℹ️ Installing generated msi: ${PKG_LATEST_NAME}"
-if ($PKG_LATEST_NAME -notlike "*.msi")
+$PKG_PATH = "${PKG_DIR}\${PKG_NAME}"
```

Comment:
joining paths is safer with https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-7.1

don't need to change it now

Reply:
Done!